cmd/soroban-cli/src/commands/network/mod.rs

```diff
@@ -52,6 +50,9 @@ pub enum Cmd {
     #[cfg(feature = "version_lt_23")]
     #[command(subcommand)]
     Container(crate::commands::container::Cmd),
+
+    /// Gets network health
```

Comment:
I think we can be more specific and indicate that this is the health of the RPC we're retrieving.
```suggestion
    /// Checks the health of the configured RPC
```